client/rpc-spec-v2/src/chain_head/event.rs

```diff
@@ -241,6 +241,85 @@ pub enum ChainHeadEvent<T> {
 	Disjoint,
 }
 
+/// The storage item received as paramter.
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct StorageQuery<Key> {
+	/// The provided key.
+	pub key: Key,
```

Comment:
Is there an advantage to `Key` be generic here (afaiu it can only be a `String`, and is used as such in the method)? It might simplify things a little to just be concrete about it?

(also it looks like `StorageResult` has a `key: String` hardcoded)

Reply:
Ok coolio, thanks for the explanation! :)